### DISCUSSION TITLE
How fully unlink Tokens Studio from all components in file?

### DISCUSSION THREAD

**Discussion by rogbjornen** [six7/figma-tokens#1780] [Q&A]
How fully unlink Tokens Studio from all components in file? If i don't want use plugin anymore?

<img width="225" alt="image" src="https://user-images.githubusercontent.com/101820672/229171647-98263e48-23f8-4243-9f0a-08390345722e.png">

**Comment by nfon-cm** [comment]
Have you tried selecting everything, then on the inspect tab, with "Deep inspect" checked, selecting all the tokens and removing them?

<img width="521" alt="image" src="https://user-images.githubusercontent.com/128799722/229549036-8bc5fd87-00aa-4e03-81f5-245468209403.png">

**Reply by rogbjornen** [reply]
Yes. But I have many pages, so I need to do it on every page.

**Comment by Danny-Cognizant** [comment]
I also want to do this but have lots of broken link tokens. When I select and remove them, the layer is affected (fill or stroke removed, font reset, etc). How do you remove broken tokens?
